packages/jest-runner/src/index.ts

```diff
@@ -149,9 +201,14 @@ class TestRunner {
           return Promise.reject();
         }
 
-        await onStart(test);
+        // Remove `if(onStart)` in Jest 27
+        if (onStart) {
```

Comment:
for later, but a PR removing `onStart`, `onResult` and `onFailure` and cleaning up the code here would be great, and we can merge it when we prepare for Jest 27. Again, no need to do it now, but might be a good idea to make that code change now while you have the context in your heads 🙂

Reply:
Yeah, I've already made a branch with these changes. As mentioned, it was easier to keep track of it.
I'll open a PR for that, when preparing for Jest 27

Reply:
ah, wonderful. Thanks! You can open the PR now and I can add it to the milestone so we don't forget 👍

Reply:
Cools, it will be the first thing to do, after this PR is finalised :smile: (so I call pull these changes on my branch before creating PR)

Reply:
Yeah, sounds like a good plan!